sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/RecognizeEntitiesBatchDocuments.java

```diff
@@ -29,8 +29,8 @@
     public static void main(String[] args) {
         // Instantiate a client that will be used to call the service.
         TextAnalyticsClient client = new TextAnalyticsClientBuilder()
-            .credential(new AzureKeyCredential("{key}"))
-            .endpoint("{endpoint}")
+            .credential(new AzureKeyCredential("184654c847d54432b8301a4b76f63045"))
```

Comment:
will remove it

Reply:
is the endpoint printed somewhere? do we need to rotate this key?